### PR TITLE
add tests for trt options

### PR DESCRIPTION
**Pull request by askhade**

### CODE REVIEW

**Comment by askhade** [comment]
@efeguney: Cant add you as reviewer here so tagging you instead.

File: qa/L0_onnxtrt_optimization/test.sh [hunk 1]
```diff
@@ -120,6 +130,12 @@ for MODEL in \
         RET=1
     fi
 
+    grep "TensorRT Execution Accelerator is set for '${MODEL}_cache_on'" $SERVER_LOG
+    if [ $? -ne 0 ]; then
+        echo -e "\n***\n*** Failed. Expected TensorRT Execution Accelerator is set\n***"
```
Comment (deadeyegoodwin) [inline qa/L0_onnxtrt_optimization/test.sh:135]: This message should say something about cache one, perhaps similar to the message 2 above

**Review by deadeyegoodwin — CHANGES_REQUESTED** [review]

**Comment by deadeyegoodwin** [comment]
@askhade  Can you also rebase to main

**Comment by deadeyegoodwin** [comment]
> 
> 
> @askhade Can you also rebase to main

I rebased... I'll start a CI run

**Review by deadeyegoodwin — APPROVED** [review]

**Pull request merged**